app/routes/quotes/components/Quote.js

```diff
@@ -45,13 +54,29 @@ export default class Quote extends Component<Props, State> {
       actionGrant,
       setDisplayAdmin,
       displayAdmin,
-      currentUser,
-      loggedIn,
-      comments,
-      deleteComment
+      emojis,
+      addReaction,
+      deleteReaction,
+      fetchEmojis,
+      fetchingEmojis
     } = this.props;
 
-    const { showComments } = this.state;
+    let mappedEmojis = [];
+    if (!fetchingEmojis) {
```

Comment:
This looks a little dirty 🗑 , but 🤷‍♂ . If this `check/map` needs to run everywhere reactions are implemented we might want to create some `util` for it, or as @finnss suggested put it into the reducer.